ttnn/cpp/ttnn/operations/ccl/shared_with_host/hetergeneous_data_structs.hpp

```diff
@@ -127,24 +127,30 @@ inline void advance_worker_global_page_interleaved (
 
     coord_t const &tensor_shape, // full tensor shape
 
-    bool &last_page_of_worker
+    bool &last_page_of_worker,
+    const uint32_t stride=1
```

Comment:
Since this appears to be called only in one place, can we just remove the default value and then also keep `last_page_of_worker` last? It's a bit of a nitpick but since last_page_of_worker is purely for debug, it'll be nice to keep it separate